app/models/ontology/class_methods_and_scopes.rb

```diff
@@ -88,7 +88,9 @@ def find_with_locid(locid, iri = nil)
 
     def find_with_iri(iri)
       locid = iri_to_locid(iri)
-      ontology = where('locid LIKE ?', '%' << locid).first
+      ontology = ::LocId.where(
+                               'locid LIKE ?', '%' << locid
+                              ).first.try(:assorted_object)
```

Comment:
Please use the agreed code style:
```ruby
method_call(arg1,
            arg2, 
            arg3, 
            arg4)
```
having the opening parenthesis on the line of the first argument and the closing one on the line of the last argument.